src/components/Common/Module/FieldItem.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <div>
+    <font-awesome-icon
+      class="float-left align-baseline m-1 mr-3 text-primary grab"
```

Comment:
Since this grab class is not referenced in draggable, you can grab by the whole fielditem and drag it that way. 
Please configure draggable so it only handles drag on `grab` element